sormas-ui/src/main/java/de/symeda/sormas/ui/clinicalcourse/HealthConditionsForm.java

```diff
@@ -80,39 +85,45 @@ protected void addFields() {
 		healthConditionsHeadingLabel.addStyleName(H3);
 		getContent().addComponent(healthConditionsHeadingLabel, HEALTH_CONDITIONS_HEADINGS_LOC);
 
-		addFields(
-			TUBERCULOSIS,
-			ASPLENIA,
-			HEPATITIS,
-			DIABETES,
-			HIV,
-			HIV_ART,
-			CHRONIC_LIVER_DISEASE,
-			MALIGNANCY_CHEMOTHERAPY,
-			CHRONIC_HEART_FAILURE,
-			CHRONIC_PULMONARY_DISEASE,
-			CHRONIC_KIDNEY_DISEASE,
-			CHRONIC_NEUROLOGIC_CONDITION,
-			DOWN_SYNDROME,
-			CONGENITAL_SYPHILIS,
-			IMMUNODEFICIENCY_OTHER_THAN_HIV,
-			CARDIOVASCULAR_DISEASE_INCLUDING_HYPERTENSION,
-			OBESITY,
-			CURRENT_SMOKER,
-			FORMER_SMOKER,
-			ASTHMA,
-			SICKLE_CELL_DISEASE,
-			IMMUNODEFICIENCY_INCLUDING_HIV);
-		TextArea otherConditions = addField(OTHER_CONDITIONS, TextArea.class);
-		otherConditions.setRows(6);
-		otherConditions.setDescription(
-			I18nProperties.getPrefixDescription(HealthConditionsDto.I18N_PREFIX, OTHER_CONDITIONS, "") + "\n"
-				+ I18nProperties.getDescription(Descriptions.descGdpr));
-
-		initializeVisibilitiesAndAllowedVisibilities();
-		initializeAccessAndAllowedAccesses();
-
-		FieldHelper.setVisibleWhen(getFieldGroup(), HIV_ART, HIV, Arrays.asList(YesNoUnknown.YES), true);
+		if (UiUtil.permitted(UserRight.SEE_SENSITIVE_DATA_IN_JURISDICTION)) {
```

Comment:
making this field inaccessible should be done similarly like you did  for the NullableOptionGroup field